registers/trillian_client/client.go

```diff
@@ -0,0 +1,81 @@
+package trillian_client
```

Comment:
Might be worth factoring this out into the Trillian repo, would help to keep the demo concise.

Reply:
Yes, it might be.